### PR TITLE
WIP User Popout Card

### PR DESCRIPTION
**Pull request by nottisa**
This is the Entire user popup card including things like description and stuff.
## Please make sure to check the following tasks before opening and submitting a PR

* [X] I understand and have followed the [contribution guide](https://github.com/revoltchat/revolt/discussions/282)
* [X] I have tested my changes locally and they are working as intended
* [X] These changes do not have any notable side effects on other Revolt projects

## TODO ##
* [ ] Finish the Popout design
    * [ ] Better text wrapping 
    * [x] Add Profile Bannner
        * [x] Make it so that if there is no banner for it to fill with color instead of a "link broken" picture  
        * [ ] Loading animation
    * [ ] Add roles field
    * [ ] Add Markdown Description
    * [ ] Add Enlarge profile button
    * [ ] Friend request/message button(s)?
    * [ ] 3 dot context menu
        * [ ] Copy User ID
        * [ ] Block user/unblock user
        * [ ] Add user to groupchat?
* [ ] Only allow one menu at a time + close as soon as not in focus.
* [ ] Theme the entire project using the theming stuffs

### CODE REVIEW

**Comment by bobthedev90368** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by bobthedev90368** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by bobthedev90368** [comment]
recheck

File: components/ui/components/floating/UserCard.tsx [hunk 1]
```diff
@@ -1,24 +1,96 @@
...
+const profiletopcontainerStyle = {
+  position: 'relative',
+  width: '100%',
+  height: '350px',
+  backgroundSize: 'cover',
+  backgroundPosition: 'center',
+  marginBottom: '20px'
+};
+
+function getStatusColor(status: string | null | undefined) {
```
Comment (oliviathevampire) [inline components/ui/components/floating/UserCard.tsx:46]: Shouldn't the colors here be changed to the ones from the current theme?

Comment (nottisa) [reply components/ui/components/floating/UserCard.tsx:46]: Was namely just place holding, I've lost intrest for now.

**Comment by insertish** [comment]
PR appears stale, closing out
NB. Further work would also have to be done to adapt to M3 design system

**Pull request closed without merging**